recipe/meta.yaml

```diff
@@ -1,17 +1,16 @@
 {% set name = "visions" %}
-{% set version = "0.7.4" %}
+{% set version = "0.7.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4f66c1da369cdbced5e1e2d2c43104caa8cfb9c694006cea1592e1673cfc1153
+  sha256: 49259f291f7737df81c4bc65e9436c43f701acb73a577393be75b26b738d5bd2
 
 build:
   number: 0
-  noarch: python
```

Comment:
Skip `py<36`, see https://github.com/dylan-profiler/visions/blob/dca86aa1f6be9d5872f7cce91414e1fa33bd1b85/setup.py#L58

Reply:
added, thank you